.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+**Description**
+Include a summary of the change, and link the associated issue (if applicable)
+List any dependencies that are required for this change.
+
+Fixes # (issue)  
```

Comment:
Instead of Fixes, maybe "Major features?"

Reply:
The "Fixes" is meant like "this PR fixes issue #().  I'll reword the line since most of the PRs pertain to issues for new features.